packages/components/src/components/TaskTree/TaskTree.js

```diff
@@ -35,8 +35,11 @@ class TaskTree extends Component {
             return null;
           }
           const { id, reason, steps, succeeded, pipelineTaskName } = taskRun;
+          const erroredTask = taskRuns.find(task => task.reason === 'Failed');
```

Comment:
probably better to check 'not succeeded' as there are many reasons that are considered failed (e.g. cancelled, timed out), see https://github.com/tektoncd/dashboard/issues/832 for some examples